### DISCUSSION TITLE
OpenBB SDK installed in Google Colab.

### DISCUSSION THREAD

**Discussion by stefanstoeckigt** [OpenBB-finance/OpenBB#3756] [Show and tell]
Guide to getting OpenBB SDK install in Google Colab.
If there is a better way, please update here.

```
%env PYTHONPATH=
````

```
%%bash
MINICONDA_INSTALLER_SCRIPT=Miniconda3-py39_4.12.0-Linux-x86_64.sh
MINICONDA_PREFIX=/usr/local
wget https://repo.continuum.io/miniconda/$MINICONDA_INSTALLER_SCRIPT
chmod +x $MINICONDA_INSTALLER_SCRIPT
./$MINICONDA_INSTALLER_SCRIPT -b -f -p $MINICONDA_PREFIX

conda install --channel defaults conda python=3.9 --yes
conda update --channel defaults --all --yes
```
```
import sys
sys.path.append("/usr/local/lib/python3.9/site-packages")
```
```
!pip install openbb
```

**Comment by jmaslek** [comment]
Hey!!  I tried this yesterday and out of the blue, you can run 

```
!pip install openbb
```
You will be met with a wall of red text but that seems to be becuase of some pre installed dependencies, but it still works.

In order to get charts you need to downgrade matplotlib to 3.5.3 or lower. I found that `!pip install matplotlib==3.2.2` does not require the runtime to be restarted, but you can install 3.5.3 and restart the runtime.

**Reply by mmistroni** [reply]
Hello i was wondering if you could help me out?
Have followed instructions here in my colab notebook (https://github.com/mmistroni/jupyter/blob/master/OpenBB.ipynb) but i am getting issues with pytorch.
Kind regards
 Marco
